src/main/java/christmas/domain/orderresult/Badge.java

```diff
@@ -0,0 +1,34 @@
+package christmas.domain.orderresult;
+
+public enum Badge {
+    SANTA("산타"),
+    TREE("트리"),
+    STAR("별"),
+    NONE("없음");
+
+    private static final int SANTA_THRESHOLD = 20000;
+    private static final int TREE_THRESHOLD = 10000;
+    private static final int STAR_THRESHOLD = 5000;
+    private final String badge;
+
+    Badge(String badge) {
+        this.badge = badge;
+    }
+
+    public static Badge valueOfBenefit(Integer totalBenefit) {
+        if (totalBenefit > SANTA_THRESHOLD) {
+            return SANTA;
+        }
+        if (totalBenefit > TREE_THRESHOLD) {
+            return TREE;
+        }
+        if (totalBenefit > STAR_THRESHOLD) {
+            return STAR;
+        }
+        return NONE;
+    }
```

Comment:
```suggestion
    public static Badge valueOfBenefit(Integer totalBenefit) {
    
        Arrays.stream(Badge.values())
                   .filter(item -> item.getThreshold() < totalBenefit)
                   .findAny( return item)
                   .orElse( return NONE);
      }
```
stream코드가 정확히 기억나지 않아 감대로 적었는데 이렇게 고칠수 있지 않을까요?
저도 똑같은 부분을 코멘트 받아.. 아쉬워 남겨봅니다